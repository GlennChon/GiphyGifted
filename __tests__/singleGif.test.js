it("<SingleGif/> - Expects", () => {
  expect(true).toBeTruthy();
});
