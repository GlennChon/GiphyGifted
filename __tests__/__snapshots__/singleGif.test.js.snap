// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<singleGif/> testing component snapshot 1`] = `
<Fragment>
  <Gif
    gif={
      Object {
        "images": Object {
          "fixed_height_downsampled": Object {
            "url": "fhdUrl",
          },
          "fixed_height_still": Object {
            "url": "fhsUrl",
          },
          "fixed_width_downsampled": Object {
            "url": "fwdUrl",
          },
          "fixed_width_still": Object {
            "url": "fwsURL",
          },
        },
      }
    }
    type="fixed-height"
  />
</Fragment>
`;
